<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Activate_DeactivateUser</class>
 <widget class="QDialog" name="Activate_DeactivateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reset Passord</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/Icon for Taskbar.png</normaloff>:/images/Icon for Taskbar.png</iconset>
  </property>
  <widget class="QGroupBox" name="Footer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>521</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#3a444c;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget1_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>251</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="forgotPasswordCancelButton">
       <property name="styleSheet">
        <string notr="true">color: #ffffff;
border:0;
height:30px;
border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/images/background.jpg);
background-position:center;
background-repeat: no-repeat;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>62</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:500;
color:#ffffff;
background: none;</string>
    </property>
    <property name="text">
     <string>Activate/Deactivate Accounts</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="lectureSearchButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#292f38;
color: #ffffff;
border:0;
height:30px;
border-top-left-radius:0px;
border-top-right-radius:5px;
border-bottom-left-radius:0px;
border-bottom-right-radius:5px;</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons White/Icons/White/Search.png</normaloff>:/icons White/Icons/White/Search.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lectureSearchLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#000;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Full Lecture Name to find their Info.</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="userInfogroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>User Information</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="activateResetButton">
       <property name="styleSheet">
        <string notr="true">background:#fbaf01;
color: #ffffff;
border:0;
height:30px;
border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Activate</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/icons White/Icons/White/Single Arrow RIGHT.png</normaloff>:/icons White/Icons/White/Single Arrow RIGHT.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disactivateResetButton">
       <property name="styleSheet">
        <string notr="true">background:#fbaf01;
color: #ffffff;
border:0;
height:30px;
border-radius:5px;</string>
       </property>
       <property name="text">
        <string>Deactivate</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/icons White/Icons/White/Single Arrow RIGHT.png</normaloff>:/icons White/Icons/White/Single Arrow RIGHT.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QRadioButton" name="studentRadioButton">
       <property name="text">
        <string>Student</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="lecturerRadioButton">
       <property name="text">
        <string>Lecturer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>51</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="Name">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Name_label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="text">
          <string>Full Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="userName">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="text">
          <string>UserName: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="userName_Label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="createAt">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="text">
          <string>Create At:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="createAt_Label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="role">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="text">
          <string>Role</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="role_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="text">
          <string>Lecturer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="Activate">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="text">
          <string>Activate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Activate_Label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="text">
          <string>Activate ? Deativate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
